Gruntfile.js

```diff
@@ -28,6 +28,25 @@ module.exports = function (grunt) {
         },
         jsonlint: {
             src: ["gpii/**/*.json", "tests/**/*.json", "testData/**/*.json"]
+        },
+        shell: {
+            options: {
+                stdout: true,
+                stderr: true,
+                failOnError: true,
+                execOptions: {
+                    maxBuffer: Infinity
```

Comment:
I played around with the ``maxBuffer`` option when I was doing the [Linux test automation](https://github.com/GPII/linux/blob/master/Gruntfile.js#L54-L58) work and felt that I had found a reasonable value. That doesn't seem to be the case for the Universal Node-based tests due to the amount of data being sent to stdout. Either ``grunt --force`` had to be used to prevent the task from failing or increasing ``maxBuffer`` again. Not having a limit means more system resources will be utilized. However in this case the workload is tackled in a VM which has its own resource limits enforced by VirtualBox so any harmful affects should technically be mitigated and not affect the host.